include/cglm/ray.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c), Recep Aslantas.
+ *
+ * MIT License (MIT), http://opensource.org/licenses/MIT
+ * Full license can be found in the LICENSE file
+ */
+
+/*
+ Functions:
+   CGLM_INLINE bool glm_line_triangle_intersect(vec3 origin, vec3 direction, vec3 v0, vec3 v1, vec3 v2, float *d);
```

Comment:
I hope we are not crossing/exceeding the 80 column here. I know we are at 2020 and this limit can be changed to 120 in the future :)

Reply:
We're at 114 here. I could add a line break somewhere if you prefer.